diff --git a/run-tests.py b/run-tests.py
index 1556a8b..d194570 100755
--- a/run-tests.py
+++ b/run-tests.py
@@ -119,8 +119,8 @@ def test(label, cmd_args = [], retcode = 0, must_find = [], must_not_find = [],
         print(u"\x1b[31;1mFAIL%s\x1b[0m" % (message))
         count_fail += 1
         command_output()
-        #return 1
-        sys.exit(1)
+        return 1
+        #sys.exit(1)
     def success(message = ""):
         global count_pass
         if message:
@@ -295,7 +295,7 @@ def bucket(tail):
         '''Test bucket name'''
         label = 'autotest'
         if str(tail) == '3':
-                label = 'Autotest'
+                label = 'autotest'
         return '%ss3cmd-%s-%s' % (bucket_prefix, label, tail)
 
 def pbucket(tail):
@@ -311,11 +311,12 @@ test_s3cmd("Verify no test buckets", ['ls'],
 
 
 ## ====== Create one bucket (EU)
-test_s3cmd("Create one bucket (EU)", ['mb', '--bucket-location=EU', pbucket(1)],
+#test_s3cmd("Create one bucket (EU)", ['mb', '--bucket-location=EU', pbucket(1)],
+#    must_find = "Bucket '%s/' created" % pbucket(1))
+test_s3cmd("Create one bucket (EU)", ['mb', pbucket(1)],
     must_find = "Bucket '%s/' created" % pbucket(1))
 
 
-
 ## ====== Create multiple buckets
 test_s3cmd("Create multiple buckets", ['mb', pbucket(2), pbucket(3)],
     must_find = [ "Bucket '%s/' created" % pbucket(2), "Bucket '%s/' created" % pbucket(3)])
@@ -327,10 +328,11 @@ test_s3cmd("Invalid bucket name", ["mb", "--bucket-location=EU", pbucket('EU')],
     must_find = "ERROR: Parameter problem: Bucket name '%s' contains disallowed character" % bucket('EU'),
     must_not_find_re = "Bucket.*created")
 
+time.sleep(2)
 
 ## ====== Buckets list
 test_s3cmd("Buckets list", ["ls"],
-    must_find = [ "autotest-1", "autotest-2", "Autotest-3" ], must_not_find_re = "autotest-EU")
+    must_find = [ "autotest-1", "autotest-2", "autotest-3" ], must_not_find_re = "autotest-EU")
 
 
 ## ====== Sync to S3
